MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TextBlock.xaml

```diff
@@ -77,4 +79,95 @@
         <Setter Property="FontSize" Value="14"/>
         <Setter Property="FontWeight" Value="Medium"/>
     </Style>
+
+    <!-- updated specification 2018 -->
+    <Style TargetType="{x:Type TextBlock}" x:Key="MaterialDesign2Body1TextBlock">
+        <Style.Resources>
+            <Style TargetType="Hyperlink" BasedOn="{StaticResource MaterialDesign2Body1Hyperlink}" />
+        </Style.Resources>
+        <Setter Property="FontSize" Value="16" />
+        <Setter Property="FontWeight" Value="Regular" />
+    </Style>
+    <Style TargetType="{x:Type TextBlock}" x:Key="MaterialDesign2Body2TextBlock">
+        <Style.Resources>
+            <Style TargetType="Hyperlink" BasedOn="{StaticResource MaterialDesign2Body2Hyperlink}" />
+        </Style.Resources>
+        <Setter Property="FontSize" Value="14" />
+        <Setter Property="FontWeight" Value="Regular" />
+    </Style>
+    <Style TargetType="{x:Type TextBlock}" x:Key="MaterialDesign2CaptionTextBlock">
```

Comment:
My personal opinion would be to drop the "2" from the key name on these. I do realize that there will be naming conflicts once that is done. However, some of these styles don't appear to be changing anything from the original style. This one for example appears to match the original `MaterialDesignCaptionTextBlock` unless I have missed something.